plugins/eslint-plugin-aws-toolkits/lib/rules/no-incorrect-once-usage.ts

```diff
@@ -122,7 +122,7 @@ export default ESLintUtils.RuleCreator.withoutDocs({
                         declaration.init.callee?.type === AST_NODE_TYPES.Identifier &&
                         declaration.init.callee.name === 'once'
                     ) {
-                        const scope = context.getScope()
+                        const scope = context.sourceCode.getScope(declaration) // TODO: should this be `getScope(node)`?
```

Comment:
@hayemaxi does this look right?

Reply:
Looks fine to me, AFAIK they are identical.
Consider:
```
L1 const x = 1
L2 const y = 2, z = 3
```
There are two VariableDeclarations here (L1, L2). L1 node.declarations is `[x = 1]`. L2 node.declarations is `[y = 2, z = 3]`.

Whether you take the root node level (L1/L2) or the individual declarations in each root, the scope is the same.

```suggestion
                        const scope = context.sourceCode.getScope(declaration)
```